torch/_inductor/codegen/wrapper.py

```diff
@@ -1721,7 +1741,7 @@ def make_buffer_allocation(self, buffer):
         return self.make_allocation(buffer.get_name(), device, dtype, shape, stride)
 
     def make_allocation(self, name, device, dtype, shape, stride):
-        if device.type in ("cpu", "cuda"):
+        if device.type in ("cpu", "cuda", self.custom_empty_strided_device):
```

Comment:
Similarly - we can do this or subclass. I'm open to either here. What do you thik ?